lib/ravelin/client.rb

```diff
@@ -14,7 +14,7 @@ class Client
     def initialize(api_key:, api_version: 2)
       @api_key = api_key
 
-      raise ArgumentError.new("api_version must be 2 or 3") unless [2,3].include? api_version
+      raise ArgumentError.new("api_version must be 2 or 3") unless [2, 3].include? api_version
```

Comment:
did you want to submit the automatic reformatting? or just the code that you changed?

Reply:
It wasn't intentional, but I figured it would be an improvement, and since there weren't too many changes I decided to leave it.

Should I revert that/do a separate commit, or is this ok?